README.md

```diff
@@ -1,7 +1,22 @@
 # pimp my knpeer
 
+## Context 
 
-This is a small React project which allow you to create and download an avatar.
+This is a small React project which allows you to create and download an avatar. It is a Toy project created during the two last KNPHackatons. 
+
+## What about?
+
+Create an app so that each knpeer can make his own personalized avatar.
+This is a project we already started at the last Hackathon. The goal is to learn react and make an app to generate avatars for KNPeers.  
+
+## Stack
+
+- React/Redux
```

Comment:
There is no Redux for the moment, we should add this specification when the PR will be merged